### PR TITLE
Initial implementation, using the command line tool "zip"

### PR DESCRIPTION
**Pull request by janvanmansum**
@GeorgiKhomeriki , please have a look.

### CODE REVIEW

File: src/main/scala/nl/knaw/dans/easy/archivebag/Main.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    if(p.waitFor() != 0) {
+      log.error("Could not create zipped bag. Zip returned: {}", p.exitValue)
+    }
+    val is = new FileInputStream(tempFile)
+    val md5 = DigestUtils.md5Hex(is)
+    is.close()
+    
+    log.info("Sending bag to deposit service ...")
+    val credsProv = new BasicCredentialsProvider
+    credsProv.setCredentials(new AuthScope("deasy", 8080), new UsernamePasswordCredentials("USER", "PASSWORD"))
```
Comment (GeorgiKhomeriki) [inline src/main/scala/nl/knaw/dans/easy/archivebag/Main.scala]: Shouldn't these hard-coded values be substituted with the variables defined in the beginning of `main`?

Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/archivebag/Main.scala]: Yep, fixed it with a new commit.

**Comment by janvanmansum** [comment]
Removed call to "zip" command. Zipping is now done in Scala code using the java.util.zip classes. @GeorgiKhomeriki please have another look.

**Comment by GeorgiKhomeriki** [comment]
Looks good to me :+1:

**Pull request merged**